{{cookiecutter.project_dashed}}-src/alembic/script.py.mako

```diff
@@ -7,6 +7,7 @@ Create Date: ${create_date}
 """
 from alembic import op
 import sqlalchemy as sa
+import sqlalchemy_utils
```

Comment:
why do you want this in the script template?  Do you use it for most of your migration scripts?

Reply:
All the special fields we use come from sqlalchemy_utils, passwords, etc. When you autogenerate a migration, Alembic will detect that we are using the fields and include the full path to the column type, but it isn't smart enough to do the import. This just saves that step.

Reply:
There is an alternate solution that patches the render of autogenerated alembic revisions rather than including these extra imports unilaterally. I can point you to a PR in the boarding app that demonstrates this.